plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.properties

```diff
@@ -7,3 +7,8 @@ flexFacetDescription = Supply App Engine Flexible Environment Functionality
 
 standardFacetName = App Engine Java Standard Environment
 standardFacetDescription = Supply App Engine Standard Environment Functionality
+
+convertToAppEngineStandardCommandDescription = Converts an Eclipse project into a Google App \
+ Engine standard environment project.
+convertToAppEngineStandardCommandName = Convert to App Engine Standard Project
+convertToAppEngineStandardMenuLabel = Convert to App Engine Standard Project
```

Comment:
Looks like the command name and the menu label are the same. Do we need two messages?